en_US/deploy/cluster/db.md

```diff
@@ -0,0 +1,77 @@
+# Embedded database
+
+EMQX uses an embedded [Mria](https://github.com/emqx/mria) database to store the following information:
+
+- routing table
+- sessions
+- configuration
+- alarms
+- etc.
+
+Mria tables are replicated across all EMQX nodes.
+It also helps with the fault-tolerance: the data is safe, as long as at least one node in the cluster is alive.
+
+If the size of the EMQX cluster is below 5 nodes, typically no tuning of the embedded database is required.
+
+However for horizontal scalability it is recommended to split the nodes in the cluster into two groups:
+
+- core nodes
+- replicant nodes
+
+## Node roles
+
+*Core nodes* serve as a source of truth for the database: they are connected in a full mesh, and each one of them contains an up-to-date replica of the data.
```

Comment:
Maybe we shouldn't mention that the replicants also are in a full mesh?  Just not a Mnesia consensus one?

Reply:
In this context I think it will be quite confusing.